Src/AutoFixture/Kernel/OmitRecursionGuard.cs

```diff
@@ -1,34 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Collections;
-
-namespace Ploeh.AutoFixture.Kernel
-{
-    public class OmitRecursionGuard : RecursionGuard
```

Comment:
I've deleted unused files that didn't build under .NET Core. The project.json includes all files in the directory and some were in the way.